wpimath/src/main/native/cpp/controller/DifferentialDriveAccelerationLimiter.cpp

```diff
@@ -14,10 +14,24 @@ DifferentialDriveAccelerationLimiter::DifferentialDriveAccelerationLimiter(
     LinearSystem<2, 2, 2> system, units::meter_t trackwidth,
     units::meters_per_second_squared_t maxLinearAccel,
     units::radians_per_second_squared_t maxAngularAccel)
+    : DifferentialDriveAccelerationLimiter(system, trackwidth, -maxLinearAccel,
+                                           maxLinearAccel, maxAngularAccel) {}
+
+DifferentialDriveAccelerationLimiter::DifferentialDriveAccelerationLimiter(
+    LinearSystem<2, 2, 2> system, units::meter_t trackwidth,
+    units::meters_per_second_squared_t minLinearAccel,
+    units::meters_per_second_squared_t maxLinearAccel,
+    units::radians_per_second_squared_t maxAngularAccel)
     : m_system{std::move(system)},
       m_trackwidth{trackwidth},
+      m_minLinearAccel{minLinearAccel},
       m_maxLinearAccel{maxLinearAccel},
-      m_maxAngularAccel{maxAngularAccel} {}
+      m_maxAngularAccel{maxAngularAccel} {
+  if (minLinearAccel > maxLinearAccel) {
+    throw std::invalid_argument(
```

Comment:
Make sure to add `@throws std::invalid_argument if minimum linear acceleration is greater than maximum linear acceleration` to the C++ docs.